parsers/IL.py

```diff
@@ -54,23 +41,62 @@ def fetch_production(zone_key="IL", session=None, target_datetime=None, logger=N
         value = re.findall("\d+", value.text.replace(",", ""))
         cleaned_list.append(value)
 
-    cleaned_list = flatten_list(cleaned_list)
+    def flatten_list(_2d_list) -> list:
+        """Flatten the list."""
+        flat_list = []
+        for element in _2d_list:
+            if type(element) is list:
+                for item in element:
+                    flat_list.append(item)
+            else:
+                flat_list.append(element)
+        return flat_list
+
+    return flatten_list(cleaned_list)
+
+
+def fetch_production(
+    zone_key="IL", session=None, target_datetime=None, logger=None
+) -> dict:
+    if target_datetime:
+        raise NotImplementedError("This parser is not yet able to parse past dates")
 
-    production = [float(item) for item in cleaned_list]
+    data = fetch_all()
+
+    production = [float(item) for item in data]
 
     # all mapped to unknown as there is no available breakdown
     return {
         "zoneKey": zone_key,
         "datetime": arrow.now(TZ).datetime,
-        "production": {
-            "unknown": production[0] + production[1]
-        },
+        "production": {"unknown": production[0] + production[1]},
         "source": IEC_URL,
         "price": PRICE,
     }
 
 
+def fetch_consumption(
+    zone_key="IL", session=None, target_datetime=None, logger=None
+) -> dict:
+    if target_datetime:
+        raise NotImplementedError("This parser is not yet able to parse past dates")
+
+    data = fetch_all()
+
+    consumption = [float(item) for item in data]
+
+    # all mapped to unknown as there is no available breakdown
+    return {
+        "zoneKey": zone_key,
+        "datetime": arrow.now(TZ).datetime,
+        "consumption": {"unknown": consumption[0]},
```

Comment:
The value of `"consumption"` should just be the number and not a breakdown-dictionary, like in this example:

```js
    {
      'zoneKey': 'FR',
      'datetime': '2017-01-01T00:00:00Z',
      'consumption': 10.0,
      'source': 'mysource.com'
    }
```

It currently fails with this error:

<img width="529" alt="Screenshot 2021-04-28 at 12 05 03" src="https://user-images.githubusercontent.com/3296643/116387173-d456b380-a81a-11eb-92ab-0c74de39a9e7.png">